mPower2/mPower2/Base.lproj/SignIn.json

```diff
@@ -39,7 +39,7 @@
                                 "type":"string",
                                 "textFieldOptions":{
                                     "keyboardType":"phonePad",
-                                    "textValidator":"^\\d{10}$",
+                                    "textValidator":"^.{10,}$",
```

Comment:
so, [this site](https://www.howtocallabroad.com/netherlands/) says NL phone numbers are nine digits (not counting the country code prefix) and also [the Wikipedia article](https://en.wikipedia.org/wiki/Telephone_numbers_in_the_Netherlands) says "Non-geographical numbers have no fixed length" (non-geographical includes mobile numbers plus some other types) so I think we may just need to abandon the idea of validating phone numbers with a regex.

Reply:
That regex I found is 10 any characters so it included the "+31" making NL 12 chars, but I agree there is really no reason to confuse things with a regex here.  I'll remove.

Reply:
Fixed and pushed